_posts/2017-12-21-14-01-netcore-beta1.md

```diff
@@ -143,9 +144,43 @@ As mentioned earlier, most of the samples for Web API OData V6.0.0 apply to Web
 the API between the two as similar as possible. While the APIs are similar, they are not identical due to differences between
 ASP.NET and ASP.NET Core, such as HttpRequestMessage is now HttpRequest.
 
-### OData V7.0.0 for ASP.NET Core 2.x
-
-There is also a new Web API OData V7.0.0 for ASP.NET available as an Alpha, which you can access by downloading the packages
-using this url: [https://www.myget.org/F/webapinetcore/api/v3/index.json](https://www.myget.org/F/webapinetcore/api/v3/index.json).
-At this point, it contains the same feature set as [Web API OData V6.0.0](https://www.nuget.org/packages/Microsoft.AspNet.OData/6.0.0)
-but uses a different namespace: Microsoft.AspNet.OData. The API is the same except for the namespace.
+##### ODataRoutingConvention
+Both `Microsoft.AspNet.OData` and `Microsoft.AspNetCore.OData` packages support same set of OData routing conventions, including default built-in routing conventions and attribute rounting convention, so that each request can be routed to matching controller for processing. All routing conventions implement the interface `IODataRoutingConvention`, however, with different definitions, as highlighted below, for the two packages due to different route matching implementations based on ASP.NET Framework and ASP.NET Core:
+- For ASP.NET Framework:
```

Comment:
Thank for updating the routing.

Do you have plan to add/update the formatter?

Reply:
I think we can add that later as needed?

---
In reply to: [196253266](https://github.com/OData/WebApi/pull/1486#discussion_r196253266) [](ancestors = 196253266)